common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js

```diff
@@ -101,8 +101,8 @@
             onSpeedChange: function(event, newSpeed, oldSpeed) {
                 this.log('speed_change_video', {
                     current_time: this.getCurrentTime(),
-                    old_speed: oldSpeed,
-                    new_speed: newSpeed
+                    old_speed: this.state.speedToString(oldSpeed),
+                    new_speed: this.state.speedToString(newSpeed)
```

Comment:
@awaisdar001 Changed both to String as on stage, both old and new speeds are in String.

Reply:
One other thing, I don't know what was the behavior previously, were both speeds were in str or float? 
Its also worth checking other video player events, like `seek`, `pause` `finish` to verify if the `position`  param if that is in str or float. 

meanwhile, if we are unsure what data type to use, we can loop in someone from data eng to confirm. 

Just being careful we don't accidentally miss config data. :)

Reply:
Previously, both were strings(verified on Stage), hence I converted them to String. For the other events, they don't save/update the speed. They use the float for time or total seconds, which is unrelated to this change.

Reply:
perfect thanks. 

Please update the sandbox and let me konw so that i can verify the change again.

Reply:
@awaisdar001 Sandbox has been updated.